pom.xml

```diff
@@ -42,16 +42,14 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
-			<scope>runtime</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.projectlombok</groupId>
-			<artifactId>lombok</artifactId>
```

Comment:
j'aurai laissé lombok pour les getter/setter faciles non ?

Reply:
yes why not, ça peut être instructif :)